projects/media-viewer/src/lib/viewers/pdf-viewer/pdf-js/pdf-js-wrapper.provider.ts

```diff
@@ -7,27 +8,30 @@ import { ToolbarEventService } from '../../../toolbar/toolbar-event.service';
 @Injectable({providedIn: 'root'})
 export class PdfJsWrapperFactory {
 
+  private linkService: pdfjsViewer.PDFLinkService;
+  private eventBus: pdfjsViewer.EventBus;
+
   constructor(
-    private readonly toolbarEvents: ToolbarEventService
-  ) {}
+    private readonly toolbarEvents: ToolbarEventService) {
+    this.linkService = new pdfjsViewer.PDFLinkService();
```

Comment:
Why do we make a new pdfjsViewer for the link service and the event bus? Shouldn't they be from the same object?

Reply:
it was previous created when the create function was called, we just moved it and stored it as a variable to be accessed in the outline component.